java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -473,6 +473,39 @@ public void setrange(BaseClient client) {
         assertTrue(exception.getCause() instanceof RequestException);
     }
 
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void getrange(BaseClient client) {
+        String stringKey = UUID.randomUUID().toString();
+        String nonStringKey = UUID.randomUUID().toString();
+
+        assertEquals(OK, client.set(stringKey, "This is a string").get());
+        assertEquals("This", client.getrange(stringKey, 0, 3).get());
+        assertEquals("ing", client.getrange(stringKey, -3, -1).get());
+        assertEquals("This is a string", client.getrange(stringKey, 0, -1).get());
+
+        // out of range
+        assertEquals("string", client.getrange(stringKey, 10, 100).get());
+        assertEquals("This is a stri", client.getrange(stringKey, -200, -3).get());
+        assertEquals("", client.getrange(stringKey, 100, 200).get());
+
+        // incorrect range
+        assertEquals("", client.getrange(stringKey, -1, -3).get());
+
+        // a redis bug lol kek
```

Comment:
Is there a Redis issue number to link to for reference so people know the status of this bug?

Also, as much as I like this comment, I don't think AWS will be okay with it upstream lol.

Reply:
Submitted https://github.com/redis/redis/issues/13207
TODO link in //

Reply:
will be fixed in redis 8

Reply:
Updated